Applications/Debugger/main.cpp

```diff
@@ -238,19 +239,3 @@ int main(int argc, char** argv)
         }
     });
 }
-
-// Magic required for linking with __cxa_demangle
-
-class A {
-public:
-    virtual ~A() {};
-    virtual int foo() = 0;
-};
-class B : public A {
-
-public:
-    virtual ~B() override;
-    int f() { return 2; }
-};
-
-B::~B() {}
```

Comment:
Please squash this change into the previous commit instead of adding it separately.
There's no need to maintain history of temporary hacks during development :)

Reply:
Sorry, I meant the previous commit *adding it*, not the previous commit in sequence :D
I should have been more clear, that's my bad!